website/src/components/Submission/FileUpload/fileProcessing.ts

```diff
@@ -108,6 +110,7 @@ export const PLAIN_SEGMENT_KIND: FileKind = {
             text: () => Promise.resolve(segmentData),
             handle: () => file,
             warnings: () => [],
+            header: () => Promise.resolve(header),
```

Comment:
Why do we need a promise here?

Reply:
as it comes from a file like text I did it the same way - why is there a Promise for text?

Reply:
`ExcelFile` has:
```typescript
    async text(): Promise<string> {
        return this.inner().text();
    }
```
where `text` returns a promise.

But all implementations of `header` simply return a value that is wrapped in a promise -> I think we could instead directly return the value.